docs/docs/Basics/Quickstart.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 4
+---
+
+## First Steps
+
+After scaffolding your project, here is some additional configuration to get your app up and running.
+
+## Install Dependencies
+
+We use pnpm in this repo so we recommend to use the same for your project. You can install it in your system in case you don't have it.
+
+```bash
+npm install -g pnpm
+```
+
+or
+
+```bash
+npm install -g @pnpm/exe
+```
+
+Then install the dependencies
+
+```bash
+pnpm i
+```
+
+## Environment Variables
+
+There's am `.env.example` file in the root of the project. Copy it to `.env` and fill in the values.
+
+```bash
+cp .env.example .env
+```
+
+For the env validation we use [T3-env](https://env.t3.gg/). When you add new environment variables, you should add the validation in the `packages/auth/env.mjs` file to make sure your app always runs with the correct env variables.
+
+## Databases
+
+Modify the `schema.prisma` file to use sqlite or any other provider of your choice
```

Comment:
might be good with a note here saying that sqlite is generally for development only, or that you should use a hosted solution like turso if you want to keep sqlite in prod